src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -121,10 +121,10 @@ protected function _hash(ConnectionInterface $db)
 /**
  * Get the key for table hashes
```

Comment:
```suggestion
 * Get the key for table hashes
 *
 * The key contains:
 * - The table name
 * - The connection name to prevent collisions across connections
 * - The fixture class name to prevent collisions when loading multiple fixtures for the same table throughout a test run
```